v4/source/users.yaml

```diff
@@ -2626,6 +3447,27 @@
             userAuth.Password = user.Password
 
             user, resp := Client.UpdateUserAuth(userID, userAuth)
+        - lang: PHP
+          source: |
+            require 'vendor/autoload.php';
+
+            use \Gnello\Mattermost\Driver;
+
+            $container = new \Pimple\Container([
+                "driver" => [
+                    "url" => "https://your-mattermost-url.com",
+                    "login_id" => "email@domain.com",
+                    "password" => "Password1",
+                  ]
+            ]);
+
+            $driver = new Driver($container);
+            $driver->authenticate();
+
+            $result = $driver->getUserModel()->updateUserAuthenticationMethod("userid", [
+                "auth_service" => "gitlab",
+                //...
```

Comment:
Unless there is a strong reason to not put all the fields, I would put all the fields in the Go example.